CHANGELOG.rst

```diff
@@ -1,5 +1,58 @@
 Changelog
 =========
+1.5.0 (2019-12-19)
+-----------------------
+MLflow 1.5.0 includes several major features and improvements:
+
+New Flavors and Flavor Updates:
+
+- New support for a LightGBM flavor (#2136, @harupy)
+- New support for a XGBoost flavor (#2124, @harupy)
+- New support for a Gluon flavor and autologging (#1973, @cosmincatalin)
+- TensorFlow.Keras and Keras parameters are now logged by ``autolog()`` (#2119, @juntai-zheng)
```

Comment:
This sounds like fix to auto-logging; it's not really a new model flavor or a model flavor update. Might be good to call out separately.